### PR TITLE
add WorldUsersProvider to admin v1

### PR DESCRIPTION
**Pull request by denisdimitrov**
Adds the worlds users provider to admin v1 to fix 💣 This will need to be added in v3 as well but there are multiple pages there and needs more thought.

### CODE REVIEW

File: src/pages/Admin/Admin.tsx [hunk 1]
```diff
@@ -52,6 +52,7 @@ import { useSelector } from "hooks/useSelector";
 import { useShowHide } from "hooks/useShowHide";
 import { useUser } from "hooks/useUser";
 import { useVenueId } from "hooks/useVenueId";
+import { WorldUsersProvider } from "hooks/users/useWorldUsers";
```
Comment (0xdevalias) [inline src/pages/Admin/Admin.tsx:55]: Can you please add an export of `WorldUsersProvider` to `src/hooks/users/index.ts` then import it like this:

```suggestion
import { WorldUsersProvider } from "hooks/users";
```

**Review by 0xdevalias — APPROVED** [review]
Small tweak that would be nice to do, otherwise LGTM! :shipit:

**Review by sunny-viktoryia — APPROVED** [review]
Approving. 🙃 

THANK YOU SO MUCH FOR QUICK FIX! ❤️

File: src/pages/Admin/Admin.tsx (hunk 1 above)
Comment (denisdimitrov) [reply src/pages/Admin/Admin.tsx]: Fixed in: https://github.com/sparkletown/sparkle/pull/1757/commits/26699fa3ad119469e8647bf1b17ea10b4d76eade

Comment (0xdevalias) [reply src/pages/Admin/Admin.tsx]: fixed in https://github.com/sparkletown/sparkle/pull/1757/commits/26699fa3ad119469e8647bf1b17ea10b4d76eade

**Pull request merged**